src/Text.Analyzers/Core (Net4)/IdentifiersShouldBeSpelledCorrectly.cs

```diff
@@ -0,0 +1,416 @@
+                                                                             LocalizableTitle,
+                                                                             LocalizableMessageTypeTypeParameterMoreMeaningfulName,
+                                                                             DiagnosticCategory.Naming,
+                                                                             DiagnosticSeverity.Warning,
+                                                                             isEnabledByDefault: false,
+                                                                             description: LocalizableDescription,
+                                                                             helpLinkUri: "https://msdn.microsoft.com/en-us/library/bb264492.aspx",
+                                                                             customTags: WellKnownDiagnosticTags.Telemetry);
+        internal static DiagnosticDescriptor MethodTypeParameterMoreMeaningfulNameRule = new DiagnosticDescriptor(RuleId,
+                                                                             LocalizableTitle,
+                                                                             LocalizableMessageMethodTypeParameterMoreMeaningfulName,
+                                                                             DiagnosticCategory.Naming,
+                                                                             DiagnosticSeverity.Warning,
+                                                                             isEnabledByDefault: false,
+                                                                             description: LocalizableDescription,
+                                                                             helpLinkUri: "https://msdn.microsoft.com/en-us/library/bb264492.aspx",
+                                                                             customTags: WellKnownDiagnosticTags.Telemetry);
+
+        private SemanticModel _model;
+        private NamingService _namingService;
+
+        public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics => ImmutableArray.Create(
+            AssemblyRule,
+            NamespaceRule,
+            TypeRule,
+            MemberRule,
+            MemberParameterRule,
+            DelegateParameterRule,
+            TypeTypeParameterRule,
+            MethodTypeParameterRule,
+            AssemblyMoreMeaningfulNameRule,
+            NamespaceMoreMeaningfulNameRule,
+            TypeMoreMeaningfulNameRule,
+            MemberMoreMeaningfulNameRule,
+            MemberParameterMoreMeaningfulNameRule,
+            DelegateParameterMoreMeaningfulNameRule,
+            TypeTypeParameterMoreMeaningfulNameRule,
+            MethodTypeParameterMoreMeaningfulNameRule
+        );
+
+        public override void Initialize(AnalysisContext analysisContext)
+        {
+            analysisContext.EnableConcurrentExecution();
```

Comment:
❗️ Instance state variables must be removed

Reply:
No longer applies